test/e2e/scenario/rum/init.scenario.ts

```diff
@@ -90,7 +90,7 @@ describe('API calls and events around init', () => {
         window.DD_RUM!.addAction('after manual view')
         window.DD_RUM!.addTiming('after manual view')
         // eslint-disable-next-line @typescript-eslint/no-unsafe-call
-        ;(window.DD_RUM as any).updateViewName('after manual view')
+        ;(window.DD_RUM as any).setViewName('after manual view')
```

Comment:
**💬 suggestion:** after properly declaring this method, the eslint-disable comment and `as any` should not be needed anymore. Remove them